gestalt-module/src/main/java/org/terasology/gestalt/module/resources/ClasspathFileSource.java

```diff
@@ -39,31 +39,39 @@
 import java.util.stream.Stream;
 
 /**
- * ModuleFileSource that exposes the content from the classpath, using a
+ * ModuleFileSource that exposes the content from the classpath, using a BSA please extend so it makes sense.
```

Comment:
```suggestion
 * ModuleFileSource that exposes content from the classpath stored under a given prefix, using an (optional) classloader.
```

I am glad I was double-checking for errors here. I think this is probably the intended meaning.

To explain more fully what this does, this class provides access to files stored on the classpath, which can be either loose files or stored within JAR files. All accessible files are listed in the `META-INF/resources` index inside of each classpath entry, in theory. This source then loads those files where the file path begins with a certain prefix (which can be empty to accept all files). A typical usage might be to use a prefix of, for example `org/terasology/engine/` to only accept files belonging to Terasology's engine module.